packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -339,7 +340,7 @@ const handleZoomResponse = async (response: Response, credentialId: Credential["
     throw Error(response.statusText);
   }
 
-  return response.json();
+  return responseClone.json();
```

Comment:
This is a dirty hack to create a new clone of the response, acceptable as a Hotfix.
Ideally, we need to send _response. But _response is sometimes of type Response and sometimes a POJO it seems.

Reply:
Can we mention in the code comments why this hack is needed?